### ISSUE TITLE
New version on add new item not display image 

### ISSUE THREAD

**Issue by juniorschen** [manijak/nativescript-carousel#170]
![image](https://user-images.githubusercontent.com/46031261/94364817-7a361a00-00a2-11eb-8158-ce81801f5f99.png)
When add new item, this create new page to swipe but when swipe:
![image](https://user-images.githubusercontent.com/46031261/94364833-8e7a1700-00a2-11eb-86e3-e0ca546d79ec.png)
Everything is empty

**Comment by jeanpaulattard** [comment]
I'm experiencing the same problem. This is even happening when creating carousel in a child component, navigate away, and navigate back to the component. The carousel items are not being recreated.

**Comment by jeanpaulattard** [comment]
@juniorschen 

I've been trying to go reason out the problem, and when I've looked into the Carousel api i noticed that there's a refresh method. What I did fixed the problem for me, so it might be worth checking it out.

In your html, get a reference to your carousel 

```
<carousel #carousel [items]="items " ... >
    <carousel-item *ngFor="let item of items">
        ....
    </carousel-item>
</carousel>
```

In your component, retrieve the instance of the carousel with view child

` @ViewChild('carousel') private carousel: ElementRef<Carousel>;`

And in your addition process, call refresh in a set timeout method (this way the refresh will be executed in another change detection cycle).

```
setTimeout(() => {
     this.carousel.nativeElement.refresh();
});
```
